src/main/kotlin/app/revanced/patches/grindr/patch/bytecode/FirebaseGetCertPatch.kt

```diff
@@ -0,0 +1,50 @@
+package app.revanced.patches.grindr.patch.bytecode
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patches.grindr.fingerprints.GetMessagingCertFingerprint
+import app.revanced.patches.grindr.fingerprints.GetReqistrationCertFingerprint
+import app.revanced.patches.grindr.Constants.SPOOFED_PACKAGE_SIGNATURE
+
+import app.revanced.patches.grindr.annotations.GrindrPatchCompatibility
+
+@Patch
+@Name("Firebase patch")
+@Description("Allows Grindr to run after being modified by ReVanced.")
+@GrindrPatchCompatibility
+
+class FirebaseGetCertPatch : BytecodePatch(
+    listOf(
+        GetReqistrationCertFingerprint,
+        GetMessagingCertFingerprint
+    )
+) {
+    override fun execute(context: BytecodeContext) {
+
+        val spoofedInstruction =
+            """
+                const-string v0, "$SPOOFED_PACKAGE_SIGNATURE"
+                return-object v0
+            """
+
+        val registrationCertMethod = GetReqistrationCertFingerprint.result!!.mutableMethod
+        val messagingCertMethod = GetMessagingCertFingerprint.result!!.mutableMethod
```

Comment:
Do not assert for null here. Handle the case that the result is null and throw fingerprint.exception.